src/routes/learn.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const router = express.Router();
+
+router.get("/learn", (req, res, next) => {
```

Comment:
In all of these routers, we don't need `next` as a parameter :)

Reply:
done